sdk/formrecognizer/tests.yml

```diff
@@ -5,3 +5,20 @@ extends:
   parameters:
     ServiceDirectory: formrecognizer
     TimeoutInMinutes: 90
+    CloudConfig:
```

Comment:
You're missing preview in SupportedClouds, so you can either add it there or remove it from this config.

Reply:
Unless the idea is to be able to enable it for testing.

Reply:
Thank you. Service only manages prod and canary. what is the idea behind preview?
For now I am removing it from the configuration

Reply:
It's another test subscription we maintain that enables some feature flags for preview features in ARM. Some services need it for certain tests.

Reply:
Ahh I see, yeah we don't need it then